layersvt/screenshot.cpp

```diff
@@ -707,6 +711,10 @@ static void writePPM(const char *filename, VkImage image1) {
     if (VK_SUCCESS != err) return;
 
     VkDevice cmdBuf = static_cast<VkDevice>(static_cast<void *>(data.commandBuffer));
+    if (deviceMap.find(cmdBuf) != deviceMap.end()) {
+        // Remove element with key cmdBuf from deviceMap so we can replace it
+        deviceMap.erase(cmdBuf);
```

Comment:
I would think that just calling erase() would be sufficient.   No need to find().